test.py

```
@@ -9094,3 +9094,1163 @@ async def test_cli_config_validate_fails_when_unconfigured(db_pool):
     finally:
         async with db_pool.acquire() as conn:
             await conn.execute("SELECT set_config('agent.is_configured', 'true'::jsonb)")
+
+
+# =============================================================================
+# ACCELERATION LAYER TESTS (Episodes & Temporal Segmentation)
+# =============================================================================
+
+
+async def test_episodes_table_structure(db_pool):
```

Ruff (0.14.10)

Redefinition of unused `test_episodes_table_structure` from line 4452

(F811)